packages/react-core/src/components/Wizard/examples/Wizard.md

```diff
@@ -161,7 +186,7 @@ class SimpleWizard extends React.Component {
         name: 'First step',
         steps: [
           { name: 'Substep A', component: <p>Substep A content</p> },
-          { name: 'Substep B', component: <p>Substep B content</p> }
+          { name: 'Substep B', component: <p>Substep B content</p>,  isDisabled: true }
```

Comment:
I'm curious what the reason was for marking this substep as disabled? I'm not saying it's a bad thing, I'm just unsure what the reasoning is.

Reply:
I wanted to cover a basic example and a case where we use a substep.

Maybe I can change the new example I created with this one to show both cases

Reply:
I think I would lean that way as well, but we also have other examples that showcase some functionality that might be considered "out of scope" of what the example is primarily showing. I'd be fine leaving it as-is for now, and at another point a description could be added in to state what the example is showcasing.